SSNFSd/ssnfsworker.cpp

```diff
@@ -171,6 +173,22 @@ QString SSNFSWorker::getPerms(QString path, qint32 uid) {
     }
 }
 
+ QByteArray SSNFSWorker::isValidPath(QByteArray path){
+    QString newPath = QTextCodec::codecForMib(1015)->toUnicode(path);
```

Comment:
`QTextCodec::codecForMib(1015)->toUnicode` is not required. Qt has a builtin `operator=` to automatically convert QByteArray to QString http://doc.qt.io/qt-5/qstring.html#operator-eq-5